backend/fms_core/resources/extraction.py

```diff
@@ -206,12 +209,15 @@ def after_save_instance(self, instance, using_transactions, dry_run):
         # Update volume and depletion status of original sample, thus recording
         # that the volume was reduced by an extraction process, including an ID
         # to refer back to the extracted sample.
+        parent_volume_left = self.extracted_from.volume - self.volume_used
```

Comment:
i would set self.extracted_from.volume in the ( if field.column_name == "Volume Used (uL)":) section above. Once we remove the volume history there won't be any reason to do it after saving. You can in the meantime use self.extracted_from.volume below instead of parent_volume_left...